src/components/ColorPicker/dataHooks.ts

```diff
@@ -0,0 +1,5 @@
+import { ColorPicker } from './ColorPicker';
```

Comment:
The idea behind using the dataHooks file, is that the driver won't import the component's react file.
So instead of having the dataHooks file import the component's react file, we need the opposite.
This file should be independent of the rest.

Reply:
updated